automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPage.scala

```diff
@@ -372,7 +372,6 @@ class NotebookPage(val url: String)(implicit override val authToken: AuthToken,
   def clickOverrideNotebookChanged(): Unit =
     if (find(confirmNotebookSaveButton).size > 0) {
       click on confirmNotebookSaveButton
-      await notVisible jupyterModal
```

Comment:
what does this do?

also can you do `git pull --rebase origin develop` and force push

Reply:
In the flow right now, there are two jupyterModals. When I was testing I noticed that even after we clicked on the confirmNotebookSaveButton, we would not get to the next step since it was awaiting for the modal underneath the first to not be visible as well. Removing this let's us get to the next step. I am a bit worried about needing a wait between the two clicks, but locally it seemed to work okay.